.vscode/launch.json

```diff
@@ -5,21 +5,24 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "integration-tests - core/common-error-conditions/incomplete-customer-details",
+            "name": "integration-tests - core/common-error-conditions/order-already-exists",
             "request": "launch",
             "runtimeArgs": [
                 "run-script",
                 "start-tests",
                 "--",
                 "--runInBand",
-                "test/features/core/common-error-conditions/implemented/incomplete-customer-details-test.js"
+                "test/features/core/common-error-conditions/implemented/order-already-exists-test.js"
```

Comment:
We should fix this https://github.com/openactive/openactive-test-suite/issues/184 next time we change this, to stop us updating this in each PR